cromwell_tools/cli.py

```diff
@@ -81,7 +81,7 @@ def add_auth_args(subcommand_parser):
     )
     submit.add_argument(
         '-i',
-        '--inputs_files',
```

Comment:
This fix, however, will very likely to break some existing scripts that rely on the CLI.